es/parse_docs.py

```diff
@@ -5,8 +5,6 @@
 
 with open(sys.argv[1], 'rb') as f:
     document = Document(f)
-    for p in document.paragraphs:
-        sys.stdout.write(p.text.encode('utf-8'))
-        sys.stdout.flush()
-    # text = '\n'.join([p.text for p in document.paragraphs])
-    # print(text.encode('utf-8'))
+    text = '\n'.join([p.text.encode('utf-8') for p in document.paragraphs])
```

Comment:
This is inefficient as it loads everything into memory and concatenate strings in a loop for no apparent reason.

If you need newlines simply `sys.stdout.write('\n')`